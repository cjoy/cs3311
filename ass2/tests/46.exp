Timothy Cappie-Wood (3363921)
Suggestions for study in 11s1

Course   Title                 UOC Requirement
-------- -----                ---- -----------
COMP2041 Software Constructio    6 Level 2 core
COMP2121 Microprocessors & In    6 Level 2 core
COMP2911 Eng. Design in Compu    6 Level 2 core
Free.... Free Electives (many   30 Free elect
GenEd... General Education (m   12 Gen Ed
