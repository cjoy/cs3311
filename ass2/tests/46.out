Timothy Cappie-Wood (3363921)
Suggestions for study in 11s1

165 167Course   Title                 UOC Requirement
-------- -----                ---- -----------
